pkg/gktest/assertion.go

```diff
@@ -0,0 +1,131 @@
+package gktest
+
+import (
+	"fmt"
+	"regexp"
+	"sync"
+
+	"github.com/open-policy-agent/frameworks/constraint/pkg/types"
+	"k8s.io/apimachinery/pkg/util/intstr"
+)
+
+// An Assertion is a declaration about the data returned by running an object
+// against a Constraint.
+type Assertion struct {
+	// Violations, if set, indicates either whether there are violations, or how
+	// many violations match this assertion.
+	//
+	// The value may be either an integer, of a string. If an integer, exactly
+	// this number of violations must otherwise match this Assertion. If a string,
+	// must be either "yes" or "no". If "yes", states that exactly one violation
```

Comment:
"yes" means that there must be at least one matching violation for the assertion to be satisfied. "no" means that there must be zero matching violations for the assertion to be satisfied"

biggest thing is the "at least one" instead of "exactly one"

Reply:
Done.